pallets/handles/src/handles-utils/src/validator.rs

```diff
@@ -27,8 +27,8 @@ fn ensure_reserved_words_canonical() {
 }
 
 /// Characters that cannot be used in the handle.
-const BLOCKED_CHARACTERS: [char; 16] =
-	['"', '#', '%', '(', ')', ',', '.', ':', ';', '<', '>', '@', '\\', '`', '{', '}'];
+const BLOCKED_CHARACTERS: [char; 17] =
+	['"', '#', '%', '(', ')', ',', '.', ':', ';', '<', '>', '@', '\\', '`', '{', '}', '/'];
```

Comment:
The test error is that this has to sorted. (see test right below this).
```suggestion
	['"', '#', '%', '(', ')', ',', '.', '/', ':', ';', '<', '>', '@', '\\', '`', '{', '}'];
```

https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=a0fbf3b89116787e84db490a012234a0